infomaniak-build-tools/macos/build-ci.sh

```diff
@@ -28,7 +28,7 @@ export QTDIR="$HOME/Qt/6.2.3/macos"
 # Uncomment to build for testing
 # export KDRIVE_DEBUG=1
 
-export MACOSX_DEPLOYMENT_TARGET="10.15"
+export MACOSX_DEPLOYMENT_TARGET="13.3"
```

Comment:
Still relevant? Should we upgrade Xcode and clang on the `github` runners instead?